authors/apps/appnotifications/urls.py

```diff
@@ -23,5 +23,10 @@
         'notifications/unsubscribe_email/<str:token>/',
         views.UnsubscribeEmailAPIView.as_view(),
         name="opt_out_link"
+    ),
+    path(
+        'notifications/delete/',
```

Comment:
why is this URL named as such `notifications/delete/`  and not just `notifications/`? I think Endpoints should be named based on the resources they interact with and not HTTP Verbs.

Reply:
Made the change @oluwagbenga-joloko